src/BasicDisplays/ModelListDisplay.tsx

```diff
@@ -0,0 +1,28 @@
+import type { ReactNode } from 'react';
+
+import type { FormDisplayChildProps } from '@/Form';
+import { cn } from '@/lib/utils';
+
+export interface ModelListDisplayProps extends FormDisplayChildProps {
+  className?: string;
+  getDisplayValue?: (value: FormDisplayChildProps['value']) => ReactNode;
+}
+
+export const ModelListDisplay = ({
+  value,
+  getDisplayValue = (val) => {
+    return val?._display_value ?? 'None';
+  },
+  className,
+  ...props
+}: ModelListDisplayProps) => {
+  const modelList = value
+    .map((val: ReactNode) => getDisplayValue(val))
+    .join(', ');
```

Comment:
Assumes getDisplayValue returns a string, which it doesn't; it returns a `ReactNode`, so `.joint` shouldn't be possible.

Reply:
Also your `.map` can just be passed the `getDisplayValue` fn for simplicity

Reply:
This sounds good, made some adjustments.